lib/rubocop/cop/performance/regexp_match.rb

```diff
@@ -199,6 +202,10 @@ def scope_root(node)
           end
         end
 
+        def modifier_if_or_unless?(source, condition)
+          source =~ Regexp.new("(if|unless) +#{Regexp.escape(condition)}\\z")
+        end
```

Comment:
Probably you can use `IfNode#modifier_form?`.
https://github.com/bbatsov/rubocop/blob/031b29b495ad435dc69172fb7603042894faae53/lib/rubocop/ast/node/if_node.rb#L76-L83

Reply:
Thanks for the advice. I fixed it.